CMake/HighFiveTargetExport.cmake

```diff
@@ -26,23 +31,18 @@ install(FILES
   DESTINATION share/${PROJECT_NAME}/CMake)
 
 
-# Generate ${PROJECT_NAME}Targets.cmake;
 # Provides IMPORTED targets when using this project from build/install trees.
 
-# NOTE: the export file is ${PROJECT_NAME}Targets to support when HighFive
-# is built within a 3rd-party project (X). Other projects can find and import
-# X-Targets.cmake containing the HighFive target
-
-# Specify targets to include in the HighFive Export
-install(TARGETS HighFive highfive_deps
-        EXPORT HighFiveTargets
-        INCLUDES DESTINATION include)
+# Specify targets to include in the HighFive Exports
+install(TARGETS _libheaders _libdeps
+        EXPORT HighFiveTargets)
 
 # Generate & install the Export for the INSTALL_INTERFACE
 install(EXPORT HighFiveTargets
-        FILE ${PROJECT_NAME}Targets.cmake
+        NAMESPACE HighFive
```

Comment:
By using a namespace exported targets are prefixed. That enables using the same `HighFiveTargetDeps.cmake` to build another deps target without risk of redefinition

Reply:
Nice!